docker/pypi/wmagent/init.sh

```diff
@@ -498,18 +519,28 @@ agent_resource_opp() {
         ## Populating opportunistic resource-control
         echo "$FUNCNAME: Populating opportunistic resource-control"
         ## Loop over the env variables starting with RESOURCE_*
-        for res in ${!RESOURCE_*}
+        for resRecord in ${!RESOURCE_*}
         do
             ## Parsing of the information stored in RESOURCE_* env variable, according to the schema reported above
-            eval `declare -p $res | sed -e "s/$res/site/g"`
-            if [[ ${site[name]} =~ .*_US_.* ]] && [[ $HOSTNAME =~ .*cern\.ch ]]; then
+            eval `declare -p $resRecord | sed -e "s/$resRecord/res/g"`
+            local resOpt=""
+            for opt in ${!res[*]}
+            do
+                if [[ ${res[$opt]} =~ true|false ]]; then
+                    ${res[$opt]} && { resOpt="$resOpt --$opt" ; continue ;}
+                else
+                    resOpt="$resOpt --$opt=${res[$opt]}"
+                fi
+            done
+
+            if [[ ${res[name]} =~ .*_US_.* ]] && [[ $HOSTNAME =~ .*cern\.ch ]]; then
                 echo "I am based at CERN, so I cannot use US opportunistic resources, moving to the next site"
                 continue
             else
-                manage execute-agent wmagent-resource-control --plugin=SimpleCondorPlugin --opportunistic --pending-slots=${site[pend]} --running-slots=${site[run]} --add-one-site=${site[name]} ; let errVal+=$?
+                manage execute-agent wmagent-resource-control $resOpt || let errVal+=$?
```

Comment:
That would be pity, since this proved to be quite useful... and the code change here was meant to only fix one error we made last time with @anpicci  - to base the data structure key names on the actual set of parameter names which the `wmagent-resource-control` script supports. So ti is actually intended to fix an overlooked part related to our code. This was not meant to serve T0 deployment needs. The fact that it could be used by the T0 team in their deployment process, came just as a side effect. .....  but ok, I'll remove this part.